playwright-e2e/pages/add-application-documents.ts

```diff
@@ -0,0 +1,44 @@
+import { type Page, type Locator, expect } from "@playwright/test";
+import config from "../settings/test-docs/config";
+import path from "path";
+
+export class AddApplicationDocuments {
+
+    readonly page: Page;
+    readonly applicationDocumentsHeading: Locator;
+    readonly addNewButton: Locator;
+    readonly typeOfDocument: Locator;
+    readonly chooseFileButton: Locator;
+    readonly giveDetailsText: Locator;
+    readonly continueButton: Locator;
+    readonly checkYourAnswersHeader: Locator;
+    readonly saveAndContinueButton: Locator;
+  
+    public constructor(page: Page) {
+      this.page = page;
+      this.applicationDocumentsHeading = page.getByRole('heading', { name: 'Application documents' });
+      this.addNewButton = page.getByRole('button', { name: 'Add new' });
+      this.typeOfDocument = page.getByLabel('Type of document');
+      this.chooseFileButton = page.locator('input#temporaryApplicationDocuments_0_document').first();
+      this.giveDetailsText = page.getByLabel('Give details of documents to follow, including why you\'re not sending them now, and when you think they\'ll be ready. (Optional)');
+      this.continueButton = page.getByRole('button', { name: 'Continue' });
+      this.checkYourAnswersHeader = page.getByRole('heading', { name: 'Check your answers' });
+      this.saveAndContinueButton = page.getByRole('button', { name: 'Save and continue' });
+    }
+
+    async uploadDocumentSmokeTest() {
+        await this.applicationDocumentsHeading.isVisible();
+        await this.addNewButton.first().isVisible();
+        await this.typeOfDocument.isVisible();
+        await this.typeOfDocument.selectOption('8: BIRTH_CERTIFICATE');
```

Comment:
May be we can parameterise selecting other options and doc types something similar like this: https://playwright.dev/docs/test-parameterize